### PR TITLE
Add support for a handful of language codes not recognised by Django/Babel

### PR DESCRIPTION
**Pull request by solarissmoke**
Refs #332.

### CODE REVIEW

**Comment by TheoChevalier** [comment]
es-XL and uk are fixed:
https://donate-wagtail-staging-pr-356.herokuapp.com/es-XL/
https://donate-wagtail-staging-pr-356.herokuapp.com/uk/

But the other ones are still not recognized, it seems:
https://donate-wagtail-staging-pr-356.herokuapp.com/ach/
https://donate-wagtail-staging-pr-356.herokuapp.com/lg/
https://donate-wagtail-staging-pr-356.herokuapp.com/lo/
https://donate-wagtail-staging-pr-356.herokuapp.com/ms/

**Comment by solarissmoke** [comment]
@TheoChevalier it seems that the compiled translations for these languages are missing on heroku - hence Django doesn't load as a valid translation. Based on the changes you've been working on recently, is this compile step something that should be happening automatically?

**Comment by TheoChevalier** [comment]
Ah, gotcha. Let me check locally

**Comment by solarissmoke** [comment]
(make sure to restart Django server after compiling translations, or it won't find them).

File: donate/settings.py [hunk 1]
```diff
@@ -275,6 +276,7 @@
     ('te', 'Telugu'),
     ('th', 'Thai'),
     ('tr', 'Turkish'),
+    ('uk', 'Ukranian'),
```
Comment (TheoChevalier) [inline donate/settings.py]: Typo, 'Ukra**i**nian'

File: donate/settings.py [hunk 2]
```diff
@@ -297,6 +299,36 @@
...
+    'bidi': False,
+    'code': 'ms',
+    'name': 'Malay',
+    'name_local': 'Malay',
+}
+django.conf.locale.LANG_INFO['uk'] = {
+    'bidi': False,
+    'code': 'uk',
+    'name': 'Ukranian',
+    'name_local': 'Ukranian',
```
Comment (TheoChevalier) [inline donate/settings.py]: Same typo, 'Ukrainian'

**Pull request merged**